.github/workflows/ci.yml

```diff
@@ -31,10 +31,13 @@ jobs:
                   node .github/actions/ci/packmode.js ${{ matrix.packmode }} > mode.json
                   cat mode.json
 
-            - run: bash ./InstanceSyncSetup.sh
-              working-directory: ./automation
-            - run: pwsh ./update-server.ps1 ${{ github.ref_name }} world backups $true $false
-              working-directory: ./automation
+            - name: cd automation && bash ./InstanceSyncSetup.sh

```

Comment:
The name is a bit missleading I think, how about `Download Mods` :D

Reply:
Well i was trying to keep the referring to the equivalent of the terminology of the server upgrade guide (aka: which step it mimicked), but i'll update it if you'd like 🤔